app/src/components/PriceLevel.js

```diff
@@ -6,26 +6,83 @@ import {Styles} from '../styles/PriceLevel';
 
 import Order from './Order';
 import {Box} from '@material-ui/core';
+import {connect} from 'react-redux';
+
+const MIN_PERCENTAGE_FACTOR_FOR_BOX_SPACE = 0.25;

```

Comment:
we currently have a constants file, I believe it should be moved to there unless that file is only for constants we plan on reusing